scripts/NextJs/configs/next.config.js

```diff
@@ -15,6 +15,8 @@ const SentryWebpackPluginOptions = {
   // recommended:
   //   release, url, org, project, authToken, configFile, stripPrefix,
   //   urlPrefix, include, ignore
+
+  silent: true, // Suppresses all logs
```

Comment:
This is a matter of opinion, but I think defaulting to giving people information (with an easy opt-out) is preferable to hiding that information by default.

```suggestion
  // Uncomment this to suppress all logs
  // silent: true, 
  
```

Reply:
Our webpack plugin debug output is very noisy and it's not giving you much info tbh (as the output is way too detailed).